TransformationSystem/scripts/dirac-production-runjoblocal.py

```diff
@@ -48,41 +48,31 @@ def __runSystemDefaults(jobID, vo):
   """
   Creates the environment for running the job and returns
   the path for the other functions.
-  
+
   """
 
-  
+
   tempdir = str(vo) + "job" + str(jobID) + "temp"
 
-  try:
-    os.mkdir(tempdir)
-    if not sys.exc_info()[1][0]:
-      S_OK("Temporary directory created.")    
-    pass
-  except:    
-    if sys.exc_info()[1][0] == 17:
-      S_OK("Temporary directory already exists.")
```

Comment:
Still my question holds: this line does absolutely nothing... What is it meant to do?